src/Features/Core/Portable/Completion/CompletionTriggerReason.cs

```diff
@@ -1,27 +0,0 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
-namespace Microsoft.CodeAnalysis.Completion
-{
-    internal enum CompletionTriggerReason
```

Comment:
Actually this whole enum seems better (though we could adjust the namign of things).